runtime/themes/spacebones_light.toml

```diff
@@ -24,57 +24,77 @@
 "comment" = { fg = "#a49da5", modifiers = ["italic"]  }
 "constant" = { fg = "#6c3163" }
 "constant.builtin" = { fg = "#6c3163", modifiers = ["bold"] }
-"string" = "#2d9574"
+"string" = "theme_aqua"
 "number" = "#6c3163"
 "escape" = { fg = "fg2", modifiers = ["bold"] }
 "label" = "#b1951d"
 "module" = "#b1951d"
 
 # TODO
-"markup.heading" = "blue"
-"markup.list" = "red"
-"markup.bold" = { fg = "yellow", modifiers = ["bold"] }
-"markup.italic" = { fg = "magenta", modifiers = ["italic"] }
-"markup.link.url" = { fg = "yellow", modifiers = ["underlined"] }
-"markup.link.text" = "red"
-"markup.quote" = "cyan"
-"markup.raw" = "green"
+"markup.heading" = "theme_blue"
+"markup.list" = "theme_red"
+"markup.bold" = { fg = "theme_yellow", modifiers = ["bold"] }
+"markup.italic" = { fg = "theme_magenta", modifiers = ["italic"] }
+"markup.link.url" = { fg = "theme_yellow", modifiers = ["underlined"] }
+"markup.link.text" = "theme_red"
+"markup.quote" = "theme_cyan"
+"markup.raw" = "theme_green"
 
-"diff.plus" = "#2d9574"
+"diff.plus" = "theme_aqua"
 "diff.delta" = "#715ab1"
 "diff.minus" = "#ba2f59"
 
-"warning" = { fg = "#da8b55" }
+"warning" = { fg = "meta" }
 "error" = { fg = "#e0211d" }
-"info" = { fg = "#b1951d" }
-"hint" = { fg = "#d1dcdf" }
+"info" = { fg = "theme_yellow" }
+"hint" = { fg = "bg2" }
 
 "ui.background" = { bg = "bg0" }
 "ui.linenr" = { fg = "bg3" }
-"ui.linenr.selected" = { fg = "#b1951d" }
+"ui.linenr.selected" = { fg = "theme_yellow" }
 "ui.statusline" = { fg = "fg1", bg = "bg2" }
 "ui.statusline.inactive" = { fg = "fg4", bg = "bg1" }
+"ui.statusline.normal" = { fg = "bg1", bg = "theme_cyan" }
+"ui.statusline.insert" = { fg = "bg1", bg = "theme_green" }
+"ui.statusline.select" = { fg = "bg1", bg = "theme_magenta" }
 "ui.popup" = { bg = "bg1" }
 "ui.window" = { bg = "bg1" }
 "ui.help" = { bg = "bg1", fg = "fg1" }
 "ui.text" = { fg = "fg1" }
 "ui.text.focus" = { fg = "fg1" }
-"ui.selection" = { bg = "bg3", modifiers = ["reversed"] }
+"ui.selection" = { bg = "hl2" }
+"ui.selection.primary" = { bg = "hl1" }
 "ui.cursor.primary" = { modifiers = ["reversed"] }
 "ui.cursor.match" = { bg = "bg3" }
+"ui.cursorline.primary" = { bg = "bg1" }
 "ui.menu" = { fg = "fg1", bg = "bg2" }
-"ui.menu.selected" = { fg = "#655370", bg = "#d1dcdf", modifiers = ["bold"] }
-"ui.virtual.whitespace" = "bg2"
+"ui.menu.selected" = { fg = "base", bg = "bg2", modifiers = ["bold"] }
+"ui.virtual" = "base-dim"
```

Comment:
The ruler is a little faint. Should it be something like

```toml
"ui.virtual.ruler" = { bg = "base-dim" }
```

so you can see it without having text cross it?

Reply:
I don't use the rulers feature, but trying it out I found `base-dim` to be too dark for my liking. I set it to `bg1` which is the same color as the cursorline, which makes it almost look like a crosshairs type thing. Not sure if that's acceptable, I can change it agian.